src/rgw/rgw_rados.cc

```diff
@@ -7001,7 +7001,7 @@ int RGWRados::stat_remote_obj(RGWObjectCtx& obj_ctx,
   int ret = conn->get_obj(user_id, info, src_obj, pmod, unmod_ptr,
                       dest_mtime_weight.zone_short_id, dest_mtime_weight.pg_ver,
                       true /* prepend_meta */, true /* GET */, true /* rgwx-stat */,
-                      &cb, &in_stream_req);
+                      false /* sync manifest */, &cb, &in_stream_req);
```

Comment:
hmm, i was thinking that stat requests wouldn't make it that far in `RGWGetObj::execute()`. but it looks like the `if (get_type() == RGW_OP_STAT_OBJ)` check comes after the manifest handling, which probably isn't what we want

i'll change this flag to `true`, and propose a change to `RGW_OP_STAT_OBJ` handling in a separate pr